src/Jackett.Common/Indexers/HDTorrents.cs

```diff
@@ -20,6 +21,16 @@ public class HDTorrents : BaseWebIndexer
     {
         private string SearchUrl => SiteLink + "torrents.php?";
         private string LoginUrl => SiteLink + "login.php";
+        private readonly Regex BannerRegex = new Regex(@"src=\\'./([^']+)\\'", RegexOptions.IgnoreCase);
+        private readonly HashSet<string> FreeleechRanks = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
Private variable naming. Update names elsewhere too.

```suggestion
        private readonly Regex _bannerRegex = new Regex(@"src=\\'./([^']+)\\'", RegexOptions.IgnoreCase);
        private readonly HashSet<string> _freeleechRanks = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```